CAFAna/Systs/RecoEnergyFDSysts.h

```diff
@@ -58,25 +67,33 @@ namespace ana
                double& weight) const override {
       
       restore.Add(sr->VisReco_NDFD,
+                  sr->RecoHadE_NDFD,
                   sr->RecoLepE_NDFD);
 
       const double scale = 0.01 * sigma;
       if (sr->isFD) {
-        if (sr->isCC && abs(sr->nuPDG) == 14) { // take away muon energy
-          sr->VisReco_NDFD += (sr->VisReco_NDFD - sr->RecoLepE_NDFD) * scale *
-            pow((sr->VisReco_NDFD - sr->RecoLepE_NDFD), 0.5);
-        }
-        else if (sr->isCC && abs(sr->nuPDG) == 12) { // fine to include electron energy
-          sr->VisReco_NDFD += (sr->VisReco_NDFD) * scale *
-            pow((sr->VisReco_NDFD), 0.5);
+        // To match LBL TDR:
+        // Whether NC or CC, Numu or Nue, we want to shift total "reconstructed neutrino energy"
+        // by the "reconstructed hadronic energy".
+        sr->VisReco_NDFD += (sr->VisReco_NDFD - sr->RecoLepE_NDFD) * scale *
+          pow((sr->VisReco_NDFD - sr->RecoLepE_NDFD), 0.5);
+        // Also shift the hadronic energy variable, if we don't do this then shifted 
+        // plots of the reconstructed hadronic energy will not be different to nominal.
+        sr->RecoHadE_NDFD += sr->RecoHadE_NDFD * scale * pow(sr->RecoHadE_NDFD, 0.5);
```

Comment:
I know I missed the boat on this one, but I would prefer `sqrt(x)` to `pow(x, 0.5)` throughout. It will be infinitesimally faster, but I also think it's much clearer to read.

Reply:
Noted for future. Will change this in next related PR.